asset/css/markbind.css

```diff
@@ -6,6 +6,10 @@ code {
     word-break: normal;
 }
 
+pre {
+    margin-bottom: 0;
+}
```

Comment:
Is there a reason why we are doing this? It is removing line spacings across many different areas of the documentation, and the missing line spacing is making the documentation rather hard to read. An example:

<img src="https://user-images.githubusercontent.com/3168908/42555793-01fd12ae-851c-11e8-823a-0a19cb75e282.png" width="450"/>